courses/machine_learning/deepdive2/feature_engineering/solutions/3_Basic_Feature_Engineering_in_Keras.ipynb

```diff
@@ -0,0 +1,2565 @@
+{
```

Comment:
<p>I wouldn't call this Accuracy since we'd reserve that for classifcation models and the formal definition of accuracy. Here I'd call it "mean squared error" instead.</p><p> </p><p>Also, I'd be careful about using the test set here? In my mind, you don't use the test set until you've done all your development of the model (architecture, hyperparameter choice, and feature engineering). Just to be on the safe side to not let evaluation of the test set influence decisions you make. You want to keep the test set truly untouched. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>